src/addons/__tests__/update-test.js

```diff
@@ -72,7 +72,10 @@ describe('update', function() {
   });
 
   it('should support apply', function() {
-    expect(update(2, {$apply: function(x) { return x * 2; }})).toEqual(4);
+    function f(x) {
```

Comment:
:+1: That worked. Strange, I could've sworn I tried that the first time around and it caused some other error. Maybe I did something else wrong.